apps/mobile/src/screens/chat/libs/components/chat-buttons/libs/components/explain-accept-task-buttons/constants/ai-response-text.constant.ts

```diff
@@ -0,0 +1,3 @@
+const AI_RESPONSE = "Way to go! I will check on you!";
```

Comment:
This constant is created 2 times. Here and in `apps/mobile/src/screens/chat/libs/components/chat-buttons/libs/components/accept-regenerate-task-buttons/constants/ai-response-text.constant.ts`